<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutPutConfig</class>
 <widget class="QWidget" name="OutPutConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="myButton" name="okbutton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="myComboBox" name="output_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.0</string>
    </property>
   </item>
  </widget>
  <widget class="myComboBox" name="output_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.0</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模拟输出2选择(mA)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模拟输出1选择(mA)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>myComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
